<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerGUI</class>
 <widget class="QMainWindow" name="ManagerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="thongTinDangNhap">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="username">
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="dangFrame">
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="dangXuatButton">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Đăng xuất</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="dangNhapButton">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Đăng nhập</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="dangKyButton">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Đăng ký</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLineEdit" name="tenDangNhap">
              <property name="placeholderText">
               <string>Tên đăng nhập</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="matKhau">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Mật khẩu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="email">
              <property name="placeholderText">
               <string>Email</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Thay đổi</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Xóa</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="CMND">
            <property name="placeholderText">
             <string>CMND</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="hoVaTen">
            <property name="placeholderText">
             <string>Họ và tên</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ngaySinh">
            <property name="placeholderText">
             <string>Ngày sinh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="congViec">
            <property name="placeholderText">
             <string>Công việc</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="gioiTinh">
            <property name="placeholderText">
             <string>Giới tính</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="thanhTimKiem">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
border-style: outset;
border-color: rgb(72, 72, 72);
background-color: #f8f9fa;
color: black;</string>
            </property>
            <property name="placeholderText">
             <string>Tìm thành viên</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="timKiemButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border-top-right-radius: 5px;
border-bottom-right-radius: 5px;
border-style: outset;
border-color: rgb(72, 72, 72);
background-color: #f8f9fa;
color: red;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/media/icons/search_bar.png</normaloff>:/media/icons/search_bar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="danhSachThanhVien">
          <property name="styleSheet">
           <string notr="true">background-color: white;
color: black;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>125</number>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>dangFrame</zorder>
   <zorder>thongTinDangNhap</zorder>
   <zorder>dangFrame</zorder>
   <zorder>dangFrame</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>dangFrame</zorder>
   <zorder>tenDangNhap</zorder>
   <zorder>matKhau</zorder>
   <zorder>email</zorder>
   <zorder>hoVaTen</zorder>
   <zorder>ngaySinh</zorder>
   <zorder>gioiTinh</zorder>
   <zorder>CMND</zorder>
   <zorder>congViec</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
